### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ltratt**
A few crates have (fortunately minor) API changes, so this PR is in little chunks. Tested briefly on b13 (and continuing to run as we speak, so it'll get a longer term test too!).

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,31 +12,31 @@ keywords = ["github", "webhook"]
...
 hex = "0.4"
-hmac = "0.10"
+hmac = "0.11"
 hyper = { version = "0.14", features = ["http1", "server", "tcp"] }
 json = "0.12"
 libc = "0.2"
-lrlex = "0.9"
-lrpar = "0.9"
-nix = "0.19"
+lrlex = { path="../grmtools/lrlex"}
```
Comment (vext01) [inline Cargo.toml:29]: do these need to be build deps if they are already normal deps?

Comment (ltratt) [reply Cargo.toml:29]: Yes, grmtools does things at compile-time and run-time.

**Comment by vext01** [comment]
Just one question.

File: Cargo.toml (hunk 1 above)
Comment (vext01) [reply Cargo.toml:29]: k

**Comment by vext01** [comment]
Ready to merge?

**Comment by ltratt** [comment]
Yep!

**Comment by vext01** [comment]
```
  failed to load source for dependency `cfgrammar`
```

**Comment by ltratt** [comment]
That's a new one on me. Maybe it's a temporary error?

bors try

**Comment by vext01** [comment]
Are your path overrides correct?

**Comment by ltratt** [comment]
Mea culpa! How did I leave that debugging stuff in?! Fixed in https://github.com/softdevteam/snare/pull/90/commits/e79f81ade26aaba55725c5b1511378c114933354. Will need squashing.

bors try

**Comment by vext01** [comment]
I thought it was intentional, so partly my bad.

**Comment by ltratt** [comment]
I can't really blame a reviewer for that... it was entirely due to my stupidity!

**Comment by vext01** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**